packages/devcmd-cli/src/index.ts

```diff
@@ -48,15 +48,42 @@ async function isDir(path: string): Promise<boolean> {
   }
 }
 
+async function startProcess(command: string, args: Array<string>, dirPath: string): Promise<number> {
+  const spawnOptions: SpawnOptions = {
+    stdio: 'inherit',
+    cwd: dirPath
+  };
+
+  const processPromise: Promise<number> = new Promise<number>((resolve, reject) => {
```

Comment:
very minor: the explicit typing for this local variable is redundant :)

Reply:
I'm so used to writing such expressive types because of the linter rules in my other projects :) thanks for pointing that out